cmd/termui/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -194,7 +194,16 @@ func (wr *WidgetsRender) prepareChainInfo(numMillisecondsRefreshTime int) {
 	case isNodeSyncingTrie:
 		syncingStr = statusSyncing
 		bytesReceived := wr.presenter.GetTrieSyncNumBytesReceived()
-		statusMessage = fmt.Sprintf("Trie sync: %d nodes, %s state size", nodesProcessed, core.ConvertBytes(bytesReceived))
+		syncPercentage := wr.presenter.GetTrieSyncProcessedPercentage()
+
+		syncPercentageOut := ""
+		if syncPercentage == 0 {
+			syncPercentageOut = statusNotApplicable
```

Comment:
The status not applicable will be shown shortly before the node reaches 1%. Please refactor this check.

Reply:
refactored here